crates/staged-sync/src/config.rs

```diff
@@ -31,10 +31,13 @@ impl Config {
         disable_discovery: bool,
         bootnodes: Option<Vec<NodeRecord>>,
         nat_resolution_method: reth_net_nat::NatResolver,
+        peers_file: Option<PathBuf>,
     ) -> NetworkConfig<ShareableDatabase<DB>> {
-        let peer_config = reth_network::PeersConfig::default()
-            .with_trusted_nodes(self.peers.trusted_nodes.clone())
```

Comment:
why was `with_trusted_nodes` removed here?

Reply:
Those config options are already set in `self.peers`, so I replaced `with_trusted_nodes` and `with_connect_trusted_nodes_only` with a call to `clone`

Reply:
ah okay, misread the this change,
this is basically `clon + change` now instead of `default + merge`

ty